recipes/pyflamegpu/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "pyflamegpu" %}
+{% set version_py = "2.0.0rc1" %}
+{% set version_cpp = "2.0.0-rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version_py }}
+
+source:
+  # Temporary, will switch to release, when conda recipe is working
+  url: https://github.com/FLAMEGPU/FLAMEGPU2/archive/refs/heads/rob_conda.tar.gz
+  # url: https://github.com/FLAMEGPU/FLAMEGPU2/archive/refs/tags/{{ version_cpp }}.tar.gz
+
+build:
+  number: 0
+  script_env:
+    - FLAMEGPU_CONDA_BUILD_THREADS
+    - FLAMEGPU_CONDA_CUDA_ARCHITECTURES
+  missing_dso_whitelist:
+    - $RPATH/libcuda.so.1  # [linux] Ignore as this comes from CUDA driver
+    - $RPATH/libnvrtc.so.12  # [linux] Conda-build can't find this in run reqs cuda-cudart
+    - $RPATH/libcudart.so.12  # [linux] Conda-build can't find this in run reqs cuda-nvrtc
+    - $RPATH/nvrtc64_120_0.dll  # [win64] Conda-build can't find this in run reqs cuda-cudart
+    - $RPATH/cudart64_12.dll  # [win64] Conda-build can't find this in run reqs cuda-nvrtc
+  ignore_run_exports:
+    - python  # [linux] Not clear why conda thinks this isn't used
+    - astpretty  # [linux, win64] Not clear why conda thinks this isn't used
+    - vc14_runtime  # [win64] Not clear why conda thinks this isn't used (we aren't static linking)
+    - ucrt  # [win64] Not clear why conda thinks this isn't used (we aren't static linking)
```

Comment:
```suggestion
```
This is not the purpose of ignore_run_exports. You should only add host packages here that you know are NOT being used at runtime. e.g. You are using a C library that has some header-only APIs.